weldx/transformations/local_cs.py

```diff
@@ -58,7 +58,9 @@ def __init__(
             provided as orientation.
             Passing a scipy.spatial.transform.Rotation object is also supported.
         coordinates :
-            Coordinates of the origin
+            Coordinates of the origin. If the passed type is a `TimeSeries` with
+            discrete values and the ``time`` parameter is set, it will be interpolated
+            the values of ``time``.
```

Comment:
We need to decide which behavior we want to have if the TimeSeries does not match the provided time (needed for time dep. orientations). Currently, the data of the time series is just interpolated, which isn't such a good solution since the LCS might be interpolated too later resulting in double interpolation. Alternatively, we can raise an exception during construction, which is probably better.

Reply:
Do we even want to allow `TimeSeries` with discrete values?

In that case it is basically just another xarray input so we should just convert (extract xarray) and use the original constructor behaviour?

Reply:
That's exactly what happens internally. The data from the `TimeSeries` is copied. Still, the problem with time is remaining.

Reply:
from looking into the code it is now basically the same behavior as before with two different xarray structures, right? (unifying time axis)
So I think just keep it that way

Reply:
Fixed as discussed offline